File: ressources/trkvz/atv.py
#! /usr/bin/python3

import requests
import json


headers={
        "Referer": "https://videojs.tmgrup.com.tr/getvideo/0fe2a405-8afa-4238-b429-e5f96aec3a5c/00000000-0000-0000-0000-000000000000"
}

s = requests.Session()
response = s.get('https://securevideotoken.tmgrup.com.tr/webtv/secure?url=https://trkvz.daioncdn.net/atv/atv.m3u8?ce=3&app=d1ce2d40-5256-4550-b02e-e73c185a314e', headers=headers).json()['Url']
print(response)

File: ressources/trkvz/php/atv.php
<?php

$url = "https://securevideotoken.tmgrup.com.tr/webtv/secure?000000&url=https%3A%2F%2Ftrkvz.daioncdn.net%2Fatv%2Fatv.m3u8%3Fce%3D3%26app%3Dd1ce2d40-5256-4550-b02e-e73c185a314e";
$headers = [
    'Origin: https://www.atv.com.tr',
    'Referer: https://www.atv.com.tr/',
    'User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36'
];

$curl = curl_init($url);

curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
curl_setopt($curl, CURLOPT_HTTPHEADER, $headers);

$response = curl_exec($curl);

// Check for errors
if (curl_errno($curl)) {
    echo 'Curl error: ' . curl_error($curl);
} else {
    // Handle the response as needed
    echo $response;
}

// Close curl session
curl_close($curl);
?>
